### ISSUE TITLE
Send message to a Player

### ISSUE THREAD

**Issue by halvors** [NOVA-Team/NOVA-Core#125]
Send message to a Player via the Player component of Entity.

I don't know excactly where this should be implemented, should this be a plugin or in NOVA-Core?

**Comment by AEnterprise** [comment]
i'm thinking in core, sending to a player is always usefull for things that change only for a specific player

**Comment by gjgfuj** [comment]
Yeah, not entirely sure how we do any special formatting or anything though. Just accept a string for now?

**Comment by RX14** [comment]
we already have networking, we can build on that

**Comment by AEnterprise** [comment]
maybe as extension of this also to everyone in a specific area/world?

**Comment by RX14** [comment]
There was an issue for a networking overhaul, but I lost it...

**Comment by RX14** [comment]
Basically make it more abstract, allowing the wrapper to just implement sending streams to places and shit, then NOVA builds a networking system on top of those streams.

**Comment by calclavia** [comment]
@RX14 Agreed

**Comment by RX14** [comment]
I think we should code it in a way where the wrapper just has to implement sending byte streams to specific places at a minimum, but minecraft for example can use netty for the packet level abstraction instead of the NOVA implementation.
